EcpBtBot/DialogFlow/Response.py

```diff
@@ -19,7 +19,7 @@ def status(self):
 class Result:
```

Comment:
it's better to inherit from `object` if you don't inherit from anyone else: class `Result(object)`